etc/profile-m-z/textroom.profile

```diff
@@ -0,0 +1,63 @@
+# Firejail profile for textroom
+# Description: Full Screen text editor heavily inspired by Q10 and JDarkRoom
+# This file is overwritten after every install/update
+# Persistent local customizations
+include textroom.local
+# Persistent global definitions
+include globals.local
+
+blacklist /usr/libexec
+blacklist ${RUNUSER}/wayland-*
+
+noblacklist ${HOME}/.config/textroom
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-proc.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.config/textroom
+whitelist ${HOME}/.config/textroom
+whitelist ${DOWNLOADS}
+whitelist /usr/share/textroom
+include whitelist-common.inc
```

Comment:
```suggestion
#include disable-xdg.inc

#mkdir ${HOME}/.config/textroom
#whitelist ${HOME}/.config/textroom
#whitelist ${DOWNLOADS}
#whitelist /usr/share/textroom
#include whitelist-common.inc
```

The PR enables the profile by default on firecfg and the program appears to be
intended to be used on arbitrary user files.

Also, I don't think much whitelisting is done for text editor profiles in
general, such as for gedit, kate and leafpad.